static/bidder-info/imds.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://{{.Host}}.technoratimedia.com/openrtb/bids/{{.Host}}"
+maintainer:
+  email: "eng-demand@imds.tv"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
```

Comment:
It is possible for two bidders to share the same user sync key (formerly called the "cookie family"). The default key is the bidder name, where imds and synacormedia are considered separate. You have three options:

- Keep the user sync keys separate. synacormedia would continue using the existing values and imds would start to build up a new identity map from scratch, separate and distinct from synacormedia. This takes extra space in the sync cookie.
- Use the imds key for both imds and synacormedia. imds would still start to build up a new identity map from scratch, but would be shared by both bidder names and use minimal space in the sync cookie.
- Use the synacormedia key for both and continue to build on the existing identity map.

We do not have a way for cleanly migrating from one user sync key to another.

Reply:
All set. Updated with changes.